Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -7,9 +7,30 @@ public static class ArrayOperations
         /// If array empty or null- don't do anything.
         /// </summary>
         /// <param name="array">Input array in a random order.</param>
+        ///
+        private static void ExchangeData(int[] array, int i, int j)
```

Comment:
ExchangeData sounds vague.
SwapElements sounds better.

Also your comments don't belong to this function, they belong to sort function.
Lastly, code should read like a book. You don't read a summary and then the title. High-level function (sort) is like a title of a book and lower-level function - step function (ExchangeData) is like a chapter. 
Please move the function down.

Reply:
I modified code per request.
Thank you!